File: demo-vk-rest-scraping.java/src/main/java/demo/model/Country.java
package demo.model;

import java.util.List;

public class Country {

    private final int cid;
    private final String title;

    private Integer citiesCount;
    private List<City> cities;

    public Country(Item item) {
        this.cid = item.getCid();
        this.title = item.getTitle();
    }

    public int getCid() {
        return cid;
    }

    public String getTitle() {
        return title;
    }

    public List<City> getCities() {
        return cities;
    }

    public void setCities(List<City> cities) {
        this.cities = cities;
        this.citiesCount = cities.size();
    }

    public Integer getCitiesCount() {
        return citiesCount;
    }

    @Override
    public String toString() {
        return "Country{" +
                "cid='" + cid + '\'' +
                ", title='" + title + '\'' +
                '}';
    }
}

File: demo-vk-rest-scraping.kotlin/src/main/kotlin/demo/model/Country.kt
package demo.model

class Country(item: Item) {

    val cid: Int
    val title: String

    var citiesCount: Int? = null
        private set

    var cities: List<City>? = null
        set(cities) {
            field = cities
            this.citiesCount = cities!!.size
        }

    init {
        this.cid = item.cid
        this.title = item.title!!
    }

    override fun toString(): String {
        return "Country{" +
                "cid='" + cid + '\'' +
                ", title='" + title + '\'' +
                '}'
    }
}
